lrz_validator/CMakeLists.txt

```diff
@@ -43,11 +43,13 @@ list(FILTER LIB_SRC_FILES EXCLUDE REGEX "main.cpp")
 
 add_executable(${PROJECT_NAME} ${HDR_FILES} "main.cpp")
 target_link_libraries(${PROJECT_NAME} PRIVATE dive_core)
+target_include_directories(${PROJECT_NAME} PRIVATE ${THIRDPARTY_DIRECTORY}/gfxreconstruct/framework)
 
 if (MSVC)
   # 4100: unreferenced formal parameter
   # 4201: prevent nameless struct/union
   target_compile_options(${PROJECT_NAME} PRIVATE /W4 /WX /wd4100 /wd4201)
+  target_link_libraries(${PROJECT_NAME} PRIVATE psapi)
```

Comment:
just curious, why do you need `psapi`?

Reply:
It is used with Windows builds in [driver_info.cpp](https://github.com/LunarG/gfxreconstruct/blob/406aa68e9881e75f14cc88da1b972999bda4c412/framework/util/driver_info.cpp#L37) and [file_path.cpp](https://github.com/LunarG/gfxreconstruct/blob/406aa68e9881e75f14cc88da1b972999bda4c412/framework/util/file_path.cpp#L34)

Reply:
> It is used with Windows builds in [driver_info.cpp](https://github.com/LunarG/gfxreconstruct/blob/406aa68e9881e75f14cc88da1b972999bda4c412/framework/util/driver_info.cpp#L37) and [file_path.cpp](https://github.com/LunarG/gfxreconstruct/blob/406aa68e9881e75f14cc88da1b972999bda4c412/framework/util/file_path.cpp#L34)

I believe I linked it to get the dive windows build to work with gfxr